app.js

```diff
@@ -29,6 +29,7 @@ app.use('/create-assignment', require('./routes/create-assignment'));
 app.use('/assignment', require('./routes/assignment'));
 app.use('/edit-assignment', require('./routes/edit-assignment'));
 app.use('/edit-user', require('./routes/edit-user'));
+app.use('/email-verification', require('./routes/email-verification'));
```

Comment:
Minor, but can you make this route /api/email-verification to signify it is not meant to be accessed by users? Also moveit away from the other routes for this same reason. I believe there is an api.js file.

Reply:
Also for doc's sake:
This page *is* supposed to be visible to users, so they can resend their email verification, which creates a new verification code.